.vscode/settings.json

```diff
@@ -4,6 +4,6 @@
     "editor.tabSize": 2,
     "editor.detectIndentation": false,
     "[typescript]": {
-      "editor.formatOnSave": true
+      "editor.formatOnSave": false
```

Comment:
@mi-xu was this setting causing trouble?

Reply:
Yes, although it was actually due to a conflict between the `.prettierrc` being configured to format everything with trailing commas, while the `tslint.json` was configured to never have trailing commas. I've changed the configs to consistently avoid trailing commas and reverted this setting instead.